cli/azd/internal/telemetry/appinsights-exporter/endpoint_config.go

```diff
@@ -0,0 +1,81 @@
+package appinsightsexporter
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+const (
+	defaultIngestionPrefix = "dc"
```

Comment:
Just checking - is this unused?   I think you might have intended to use it on L49?